seed/challenges/02-javascript-algorithms-and-data-structures/es6.json

```diff
@@ -75,33 +75,32 @@
         "<code>i</code> is not defined because it was not declared in the global scope. It is only declared within the for loop statement. <code>printNumTwo()</code> returned the correct value because three different <code>i</code> variables with unique values (0, 1, and 2) were created by the <code>let</code> keyword within the loop statement.",
         "<hr>",
-        "Fix the code so that each camper returns their correct index when they are called.",
-        "<strong>Note</strong><br>Remember to add <code>\"use strict\";</code> to the top of your code."
+        "Fix the code so that <code>i</code> declared in the if statement is a separate variable than <code>i</code> declared in the first line of the function. Be certain not to use the <code>var</code> keyword anywhere in your code.",
+        "<strong>Note</strong><br>Remember to add <code>\"use strict\";</code> to the top of your code.",
+        "This exercise is designed to illustrate the difference between how <code>var</code> and <code>let</code> keywords assign scope to the declared variable. When programming a function similar to the one used in this exercise, it is often better to use different variable names to avoid confusion."
       ],
       "challengeSeed": [
-        "var newCampers = [{camper: \"Wil\"}, {camper: \"Sam\"}, {camper: \"Dav\"}];",
-        "// only change code below this line",
-        "for (var i = 0; i < newCampers.length; i++) {",
-        "// only change code above this line",
-        "  newCampers[i].roleCall = function() {",
-        "    return \"Camper # \" + (i + 1) + \" has arrived.\";",
-        "  };",
+        "function checkScope() {",
+        "  var i = \"function scope\";",
+        "  if (true) {",
+        "    i = \"if statement scope\";",
+        "    console.log(i);",
+        "  }",
+        "  console.log(i);",
         "}",
-        "// test your code",
-        "console.log(newCampers[0].roleCall());",
-        "console.log(newCampers[1].roleCall());",
-        "console.log(newCampers[2].roleCall());"
+        "// only change the code above this line",
+        "checkScope();"
       ],
       "tests": [
-        "assert(newCampers[0].roleCall() === \"Camper # 1 has arrived.\", 'message: <code>newCampers[0].call()</code> should call the index of the first camper');",
-        "assert(newCampers[1].roleCall() === \"Camper # 2 has arrived.\", 'message: <code>newCampers[1].call()</code> should call the index of the second camper');",
-        "assert(newCampers[2].roleCall() === \"Camper # 3 has arrived.\", 'message: <code>newCampers[2].call()</code> should call the index of the third camper');"
+        "// Test that <code>var</code> keyword is not used",
+        "// Test that <code>i</code> in if statement logs \"if statement scope\"",
```

Comment:
These regex expressions are great! Thanks for the help and the explanation. And I agree... I will rename 'if statement scope to 'block scope'. Also, the regex statements work fine without any additional escaping when testing locally.